src/pages/login-form.scss

```
@@ -0,0 +1,40 @@
+.login {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.login-form{
+  min-height: 460px;
+  width: 340px;
+  padding: 50px 30px 18px 30px;
+  box-shadow: 0px 0px 6px rgba(0, 0, 0, 0.14);
```

HEX тоже позволяет задействовать альфа-канал. Можно использовать его для единообразия
```suggestion
  box-shadow: 0px 0px 6px #00000014;
```